modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/NewContentDialog.ts

```diff
@@ -60,8 +65,17 @@ module app.create {
             });
 
             this.contentList = new app.create.NewContentDialogList();
+            this.contentList.addClass("new-content-types-list");
```

Comment:
I Can't find any need for "new-content-types-list" adding. We can just use default 'content-types-list' for getElement operation.